src/util/defined.js

```diff
@@ -0,0 +1,11 @@
+// Checks that passes properties are 'defined', meaning that calling them with d, i returns non null values
+export default function(d, i) {
+    for (var c = 0, j = arguments.length; c < j; c++) {
+        if (arguments[c] !== d && arguments[c] !== i) {
```

Comment:
I am trying to do this:

`// Returns function that checks if passes properties are 'defined', meaning that calling them with (d, i) returns non null values
export function defined() {

    var self = this;

    return function(d, i) {
        for (var c = 0, j = self.arguments.length; c < j; c++) {
            if (self.arguments[c](d, i) === null) {
                return false;
            }
        }
    };
}`

But eslint stops me from `var self = this;` any suggestions?